src/frontend/komponenter/Fagsak/Behandlingsresultat/EøsPeriode/fellesKomponenter.tsx

```diff
@@ -38,14 +44,24 @@ export const EkspandertTd = styled(Table.DataCell)`
 
 interface IEøsPeriodeSkjemaContainerProps {
     maxWidth?: number;
+    lesevisning: boolean;
+    status: EøsPeriodeStatus;
 }
 
 export const EøsPeriodeSkjemaContainer = styled.div`
     max-width: ${(props: IEøsPeriodeSkjemaContainerProps) =>
         props.maxWidth ? `${props.maxWidth}rem` : '30rem'};
-    border-left: 0.0625rem solid var(--navds-global-color-orange-500);
+    border-left: 2px solid
+        ${(props: IEøsPeriodeSkjemaContainerProps) => {
+            if (props.lesevisning) return NavdsSemanticColorBorderMuted;
+            if (props.status === EøsPeriodeStatus.IKKE_UTFYLT)
+                return NavdsSemanticColorFeedbackWarningBorder;
+            if (props.status === EøsPeriodeStatus.UFULLSTENDIG)
+                return NavdsSemanticColorFeedbackDangerBorder;
+            return NavdsSemanticColorInteractionPrimary;
+        }};
     padding-left: 2rem;
-    margin-left: -2rem;
+    margin-left: -3rem;
```

Comment:
Dette er gjort for at kantfargen skal ligge på linje med statusikonet over. Se skjermbildene for kantfarge ☝️